src/components/NewsletterModal/NewsletterModal.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import Newsletter from './Newsletter'
+import Modal from '../Modal/Modal'
+import { ModalProps } from '../Modal/types'
+
+export default function NewsletterModal({ onClose, isOpened }: ModalProps) {
+    return (
+        <Modal isOpened={isOpened} onClose={onClose}>
+            <Newsletter />
```

Comment:
may we also dinamically set the children to use, instead of creating a component for every modal ?

Reply:
yes it we'll be great.